lighthouse-core/config/default-config.js

```diff
@@ -5,29 +5,19 @@
  */
 'use strict';
 
-const Driver = require('../gather/driver');
-const emulation = require('../lib/emulation').settings;
+const constants = require('./constants');
 
 /* eslint-disable max-len */
 
 module.exports = {
-  settings: {
-    maxWaitForLoad: Driver.MAX_WAIT_FOR_FULLY_LOADED,
-    throttlingMethod: 'devtools',
-    throttling: {
-      requestLatencyMs: emulation.MOBILE_3G_THROTTLING.adjustedLatencyMs,
-      downloadThroughputKbps: emulation.MOBILE_3G_THROTTLING.adjustedDownloadThroughputKbps,
-      uploadThroughputKbps: emulation.MOBILE_3G_THROTTLING.adjustedUploadThroughputKbps,
-      cpuSlowdownMultiplier: emulation.CPU_THROTTLE_METRICS.rate,
-    },
-  },
+  settings: constants.defaultSettings,
```

Comment:
I'd be against that since it feels like a small when at the cost of manual/error-prone work to copy properties from one file to the other and muddies the source of truth, the shape of settings is a cmd+click away still :)

Maybe it's just me, @paulirish WDYT?

Reply:
at first i was like "yah" but now i'm like "nah"

having the keynames present would be nice, but mostly a tease if the actual values aren't there. might as well be clear that the source of truth is not this file.

so i'm +1 on keeping it as is in the PR